build.gradle

```diff
@@ -178,6 +182,32 @@ task perfJar(type: Jar) {
     with jar
 }
 
+task jmh(type: JavaExec, description: 'Executing JMH benchmarks', group: 'performance') {
```

Comment:
*sideshow bob rake noises*. This is probably fine, but I'm the last person to ask, gradle is completely greek to me.

Reply:
It's a task, so you can `./gradlew jmh` to run the benchmarks.
It will execute some Java, specifcally the main mentioned in the body.
A nice description for when you wonder what the tasks do, and a group so that all performance-related tasks are grouped together.

Simple!